src/libs/ReportUtils.ts

```diff
@@ -2612,7 +2613,8 @@ function getGroupChatName(participants?: SelectedParticipant[], shouldApplyLimit
             .sort((first, second) => localeCompare(first ?? '', second ?? ''))
             .filter(Boolean)
             .join(', ')
-            .slice(0, CONST.REPORT_NAME_LIMIT);
+            .slice(0, CONST.REPORT_NAME_LIMIT)
+            .concat(shouldAddEllipsis && shouldApplyLimit ? '...' : '');
```

Comment:
Adding the ellipsis should be controlled solely by `shouldAddEllipsis`. Let's change its definition to:

```js
shouldAddEllipsis = shouldApplyLimit && length > limit
```